### PR TITLE
Minor fixes to typos. Simplified some language in line with accessibi…

### PR DESCRIPTION
**Pull request by zarbspace**
…lity guidance. Removed references to BPI.

### CODE REVIEW

File: Reproducible-Analytical-Pipelines-MVP.md [hunk 1]
```diff
@@ -2,10 +2,10 @@
 
 ## Things to be aware of
 
-This guidance has been agreed upon by a working group of the [Reproducible Analytical Pipelines Champion](https://gss.civilservice.gov.uk/about-us/champion-networks/reproducible-analytical-pipeline-rap-champions/).
-It will be reviewed periodically by the RAP Champions and the Best Practice and Impact data science team.
+This guidance has been agreed by a working group of the [Reproducible Analytical Pipelines Champions](https://gss.civilservice.gov.uk/about-us/champion-networks/reproducible-analytical-pipeline-rap-champions/).
+It will be reviewed periodically by the RAP Champions and the Analysis Standards and Pipelines data science team in Methods and Quality Directorate the Office for National Statistics.
```
Comment (rowanhemsi) [inline Reproducible-Analytical-Pipelines-MVP.md]: Should this be *of* or *at* the Office for National Statistics? Also, is it Methods and Quality or Methodology and Quality?

File: Reproducible-Analytical-Pipelines-MVP.md [hunk 2]
```diff
@@ -15,16 +15,16 @@ Producers of analysis may incorporate some or all of these guidelines into their
...
 
 ### Who is this guidance for?
 
 This guidance is for official statisticians and analysts working in the UK government.
-This guidance may also be helpful for users of official statistics.
+This guidance may also be helpful for users of official statistics or other government analysis.
 
-This guidance has been developed by the Best Practice and Impact division at the Office for National Statistics.
-The Best Practice and Impact division supports government analysis by providing guidance, consultancy and training.
+This guidance has been developed by the Analysis Standards and Pipelines team in the Quality and Improvement Division at the Office for National Statistics.
```
Comment (rowanhemsi) [inline Reproducible-Analytical-Pipelines-MVP.md]: MQD is mentioned above rather than QID - should we stick to one?

**Review by rowanhemsi — COMMENTED** [review]
Definitely an improvement to the language, just a couple of comments relating to our directorate/division

**Comment by zarbspace** [comment]
It should be at the Office for National Statistics, sorry.  And Methodology and Quality, which is a shame, because it’s more complicated…

Dr Martin Ralphs (he/him) | Head of Analysis Standards and Pipelines | Methodology and Quality |
Office for National Statistics | Swyddfa Ystadegau Gwladol
w: 0207 592 8650 m: 07760 314213 | ***@***.******@***.***> | www.ons.gov.uk/<https://www.ons.gov.uk/> | @ons<https://twitter.com/ONS>

We work flexibly at ONS. I sent this email at a time that suited me, I don’t expect you to read it or reply outside of your working hours.

I work compressed hours. Friday is my non-working day, although I do check my email periodically.

**Review by rowanhemsi — APPROVED** [review]

**Pull request merged**